Formula/arangodb.rb

```diff
@@ -93,5 +116,17 @@ def plist; <<~EOS
     testcase = "require('@arangodb').print('it works!')"
     output = shell_output("#{bin}/arangosh --server.password \"\" --javascript.execute-string \"#{testcase}\"")
     assert_equal "it works!", output.chomp
+
+    start_cmd = "#{bin}/arangodb start --starter.instance-up-timeout 1m --starter.mode single"
+    ohai start_cmd
+    stdout, stderr, status = Open3.capture3(start_cmd)
```

Comment:
We need to have `ensure` block, so if there is something wrong we do not keep the running server on CI.

Reply:
@igas what is the preferred way to kill a process in a formula?